src/components/Extension/ShellyExtension/types.ts

```diff
@@ -0,0 +1,19 @@
+export {
+  type DataPoints,
+  useCompleteExtensionActivity,
+} from '../../../hooks/useCompleteExtensionActivity'
+
+export type ShellyExtensionSettings = {
+  apiKey?: string
+  environment: 'production' | 'staging'
+}
```

Comment:
This can be removed:
- These are not the correct settings
- We don't need the settings in the front-end anyway